andela/ah-scorpion-frontend
# #160179368 Add Homepage

#### What does this PR do?
- Adds a homepage that is accessible to all users

#### Description of tasks completed
- Added a `Home.js` component to contain the main content body for homepage
- Added a `HomeFooter.js` component for the homepage footer
- Added a `Home.js` container to hold homepage components
- Added a default route for homepage in `App.js`
- Added a custom css class for the home container
- Added assets folder to contain images

#### How can this PR be manually tested
- pull from the branch `ft-home-page-60179368`
- run `npm install` to install dependencies
- run `npm start -s` to start the server
- in the browser tab opened, the page for `localhost:3000` should display a homepage for Authors Haven similar to the one [here](https://8ltxuwr5u9kgha8qdfknag-on.drv.tw/ah-scorpion-templates/)
- the `sign up` button should navigate you to the signup page.

#### What are the related PR stories
- [#160179368](https://www.pivotaltracker.com/story/show/160179368)
